src/domain/mainwindow.py

```diff
@@ -401,23 +404,34 @@ def save_config_to_file(self):
         """Method to save configuration to file."""
 
         logger.info("Saving configuration to file...")
-        # Serializing cameras per class type
+        # Prepare serialization for cameras per class type
         cameras_to_dict = []
         for camera in cameras:
             if (
                 camera.type == Camera.CameraTypes.USBCamera
                 or camera.type == Camera.CameraTypes.FTPCamera
             ):
                 cameras_to_dict.append(camera.serialize())
-        # Serialize configured notification methods
+        # Prepare serialization for notifiers per class type
         notification = {}
         for key in Notifier.notifiers:
-            notification[key] = Notifier.notifiers[key].serialize()
+            if key and Notifier.notifiers[key]:
+                notification[key] = Notifier.notifiers[key].serialize()
+        # Prepare serialization for actuators per class type
+        actuators = []
+        for key in Actuator.actuators:
+            if (
+                Actuator.actuators[key]
+                and Actuator.actuators[key].type == Actuator.ActuatorTypes.ROADSIGN
+            ):
+                actuators.append(Actuator.actuators[key].serialize())
```

Comment:
actuators = [value.serialize()
            for key, value in Actuator.actuators.items()
            if value and value.type == Actuator.ActuatorTypes.ROADSIGN]